lib/gplt.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require "thor"
+require "./tests/helpers/perf_run_helper"
+
+# Add pry support when available
+begin
+  require "pry"
+rescue LoadError # rubocop: disable Lint/HandleExceptions
+  # do nothing
+end
+
+module GPLT
+  class Error < StandardError; end
+
+  # Class to define CLI subcommands
+  class CLI < Thor
+    extend PerfRunHelper
+
+    default_test_type = "apples to apples"
+
+    option :test_type, default: default_test_type
```

Comment:
Are these options of the `gplt` command or the `validate2baseline` sub-command? If the former, does this mean that the options for all sub-commands need to be options of the primary command? Based on the discussion on this being the main CLI for GPLT it would seem best to limit the scope for options to each sub-command to avoid collisions / cluttering the primary 'namespace' but I'm not sure if that is possible.